snoop/data/models.py

```diff
@@ -882,3 +882,27 @@ def __str__(self):
     class Meta:
 
         verbose_name_plural = 'statistics'
+
+
+class Thumbnail(models.Model):
+    """Database model for storing the Thumbnail corresponding to a Digest.
+    """
+
+    class Size(models.IntegerChoices):
+        SMALL = 100
+        MEDIUM = 200
+        LARGE = 400
+
+    original = models.ForeignKey(
```

Comment:
Again, why is this needed? you can pull the Digest from the `blob` with no issues.

Let's not complicate the schema any more than we need to.

You probably want to take a look at these: https://en.wikipedia.org/wiki/Database_normalization 2NF 3NF